hk-modding-templates.md

```diff
@@ -0,0 +1,34 @@
+# Hollow Knight Mod Templates (NuGet)
+
+This is a NuGet package for making new hollow knight mods. [See it on the NuGet Gallery](https://www.nuget.org/packages/HKModding.HKMod.Templates/1.0.14)
+If you are looking for the IDE Template, [this](https://prashantmohta.github.io/ModdingDocs/your-first-mod.html#installing-the-mod-template) might be it.
```

Comment:
I think you should move this line to be the first sentence as somewhat of a warning. Cuz using template from nuget is not as easy as using Vs template.

So maybe it could be like Note: If you use an IDE like Visual Studio, you should use the IDE template found here (link). Only use this if you are comfortable with using nuget templates